website/docs/components/icon/partials/guidelines/overview.md

```diff
@@ -0,0 +1,10 @@
+## What is an icon
+
+An icon is used to provide visual flourish to text, and in some rare contexts meaning on its own.
+
+In the context of the HDS components there is an important distinction to make:
+
+- in Figma the term "icon" can be used interchangeably to indicate the Figma component and the visual element itself (or "glyph")
+- in code the "icon" as glyph is nothing more than an `<svg>` HTML element, while the `Hds::Icon` is the actual Ember component that renders the `<svg>` element.
+
+You can see all the available icons ("glyphs") in [our icon library](/icons/library), also known as Flight Icons library.
```

Comment:
I don't think we need to keep repeating the alternate "glyph" name. I would just mention it once. I would think the majority of people would use & understand the term "icon" vs. "glyph" anyway.

Reply:
It's now removed above